app/views/home/index.en.html.erb

```diff
@@ -1,23 +1,73 @@
-<% title '' %>
+<% title 'Check when your criminal record expires - GOV.UK' %>
```

Comment:
The suffix `- GOV.UK` is added automatically, no need to provide it here.

Reply:
Actually, this should be left blank (as before) and change the locale key `en.service.name` to whatever the service name is.